nexus/src/app/disk.rs

```diff
@@ -54,6 +54,17 @@ impl super::Nexus {
                         ),
                     });
                 }
+
+                // Reject disks where the block size isn't at least
```

Comment:
To close out https://github.com/oxidecomputer/omicron/issues/1029 , could we do a granularity check too?

(I think it's okay to be slightly redundant and have this be a separate check from the block device size)

Reply:
I'm not familiar with granularity checks. Can you explain this to me or point me in the right direction?

Reply:
Got it! Makes sense now. Thanks for the clarification.